ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -20,6 +20,15 @@ const (
 	FailedToUpdateService            ServiceStatus = "FailedToUpdateService"
 )
 
+type RayServiceUpgradeStrategy string
+
+const (
+	// During upgrade, NewCluster strategy will create new upgraded cluster and switch to it when it becomes ready
+	NewCluster RayServiceUpgradeStrategy = "NewCluster"
+	// Having None as the strategy will mean down-time while it wait for new cluster to be ready when performing upgrade
```

Comment:
IIUC, if the upgrade strategy is set to None, no new RayCluster will be created.

Reply:
Yes, my fault, deleted the wrong thing while playing around with few things.